src/routes/auth/forgot-password/+page.svelte

```diff
@@ -2,18 +2,20 @@
 	import { Input, Label, Button, A } from 'flowbite-svelte';
 	import i18n from '$lib/i18n';
 	import { addToast } from '$lib/toaster.svelte.js';
+	import { page } from '$app/stores';
 
 	const { data } = $props();
 	const { supabase } = data;
 
 	let email = $state('');
+	const redirect = $derived($page.url.searchParams.get('redirect') || '/');
```

Comment:
```suggestion
	const redirect = $derived($page.url.searchParams.get('redirect') || '/auth');
```
On veut pas rediriger sur le auth après un forgot_password ?

Reply:
pareil

Reply:
ici ça fait du sens, je le change 🙂